docs/source/depth_estimation.mdx

```diff
@@ -0,0 +1,198 @@
+# Depth estimation
+
+Depth estimation datasets are used to train a model to approximate the relative distance of every pixel in an
+image from the camera aka depth. The applications enabled by these datasets mostly lie in areas like visual machine
+perception and perception in robotics. This guide will show you how to apply transformations
+to a depth estimation dataset.
+
+Before you start, make sure you have up-to-date versions of `albumentations` installed:
+
+```bash
+pip install -U albumentations 
+```
+
+[Albumentations](https://albumentations.ai/) is a Python library for performing data augmentation
+for computer vision. It supports various computer vision tasks such as image classification, object
+detection, segmentation, and keypoint estimation.
+
+This guide uses the [NYU Depth V2](https://huggingface.co/datasets/sayakpaul/nyu_depth_v2) dataset which is 
+comprised of video sequences from a variety of indoor scenes as recorded by both the RGB and Depth cameras
+from the Microsoft [Kinect](http://www.xbox.com/kinect). The dataset consists of scenes from 3 cities and provides images along with
+their depth maps as labels.
+
+Load the `train` split of the dataset and take a look at an example:
+
+```py
+>>> from datasets import load_dataset
+
+>>> train_dataset = load_dataset("sayakpaul/nyu_depth_v2", split="train")
+>>> index = 17
+>>> dataset[index]
+{'image': <PIL.PngImagePlugin.PngImageFile image mode=RGB size=640x480>,
+ 'depth_map': <PIL.PngImagePlugin.PngImageFile image mode=L size=640x480>}
+```
+
+The dataset has three fields:
+
+* `image`: a PIL image object.
+* `depth_map`: depth map of the image.
+
+Next, check out an image with:
+
+```py
+>>> train_dataset[index]["image"]
+```
+
+<div class="flex justify-center">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/datasets/depth_est_sample.png">
+</div>
+
+Similarly, you can check out the respective depth map:
```

Comment:
Probably better if we tell the user to look at the depth map instead of suggesting it

```suggestion
Now take a look at its corresponding depth map:
```